Branch-SDK/Branch-SDK/BNCLog.m

```diff
@@ -472,12 +473,19 @@ BNCLogOutputFunctionPtr _Nullable BNCLogOutputFunction() {
     }
 }
 
+void BNCLogCloseLogFile() {
```

Comment:
I use it in the unit tests and jdee uses it in his tests too.  It explicitly flushes closes log files.